examples/basics/visualising_open_industrial_data.ipynb

```diff
@@ -0,0 +1,436 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "view-in-github"
+   },
+   "source": [
+    "<a href=\"https://colab.research.google.com/github/cognitedata/cognite-python-docs/blob/visualizing-open-industrial-data/examples/basics/visualising_open_industrial_data.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "eAXnMj5E9jWG"
+   },
+   "source": [
+    "# Visualising Open Industrial Data\n",
+    "\n",
+    "## What this notebook will achieve\n",
+    "\n",
+    "* Extract live data from an oil rig in the North Sea.\n",
+    "\n",
+    "* Visualize and inspect data directly from the Cognite Data Fusion.\n",
+    "\n",
+    "\n",
+    "\n",
+    "## Getting started\n",
+    "\n",
+    "* Having a basic understanding of Python concepts will help to understand the process.\n",
+    "\n",
+    "* Cognite has released *live* data to the public on Cognite Data Fusion streaming from [Valhall](https://www.akerbp.com/en/our-assets/production/valhall/), one of Aker's oil fields.\n",
+    "\n",
+    "* To access the data, generate an API Key on [Open Industrial Data](https://openindustrialdata.com/). Get your key via the Google Access platform. You will be asked to fill out some personal information to generate your personal key.\n",
+    "\n",
+    "* Visualize some of the machines (assets) on Valhall with Cognite's [Operational Intelligence](https://opint.cogniteapp.com/publicdata/infographics/-LOHKEJPLvt0eRIZu8mE) dashboard. This data on this page shows is streaming live data from the Valhall oil field located in the North Sea.\n",
+    "\n",
+    "* To understand how to interact with the data using the Python SDK ([Docs](https://cognite-sdk-python.readthedocs-hosted.com/en/latest/)) follow along in this notebook."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "k8-Oj493ReNU"
+   },
+   "source": [
+    "## Environment Setup"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "4Lf5pnaDBLnc"
+   },
+   "source": [
+    "#### Install the Cognite SDK package"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "pbgkP3p59iiK"
+   },
+   "outputs": [],
+   "source": [
+    "!pip install cognite-sdk\n",
```

Comment:
Instead of dynamically installing these dependencies with pip, you should just refer to dependency installation instructions in the top level readme. There the user is instructed to install these packages (and the other deps you are missing like scikit-learn) to his environment.

Reply:
The reason the installations are in the notebook is because we want it to be possible to open this notebook in colab and run through it without doing any changes. When Que and I used this at the WiDS conference we found out that it only ran if we upgraded numpy. Also cognite-sdk needs to be installed. Other dependencies, like scikit-learn is not necessary to install in order for the notebook to run in colab.

Reply:
I see. Maybe add a note about that then, stating that it only needs to be run in colab. And no need to show the output for this cell.

Reply:
Good point. I added a note